logging/overlays/nerc-ocp-obs/kustomization.yaml

```
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- ../../base
+
+patchesStrategicMerge:
+  - externalsecrets/openshift-logging-lokistack-gateway-bearer-token_patch.yaml
```

`patchesStrategicMerge` is deprecated syntax. This should read:

```suggestion
patches:
  - path: externalsecrets/openshift-logging-lokistack-gateway-bearer-token_patch.yaml
```

---

Thank you! Is there a way to detect deprecated syntax by some tool/command?

---

Just use a newer version of Kustomize:

```
$ kustomize version
v5.2.1
$ kustomize build logging/overlays/nerc-ocp-infra/ > /dev/null
# Warning: 'patchesStrategicMerge' is deprecated. Please use 'patches' instead. Run 'kustomize edit fix' to update your Kustomization automatically.
```

---

Thanks. I changed all occurrences here.